src/lib/graphqlTimeoutMiddleware.ts

```diff
@@ -0,0 +1,49 @@
+import { IMiddleware } from "graphql-middleware"
```

Comment:
👋 hi middleware. IMatt

Reply:
Yeah this is a very ugly (IMO) TS convention that indicates this is an interface. 🤷‍♂️